app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/$portName-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libplist$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:plistutil$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libglib2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libplist${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	devel:libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES_devel="
+	libplist$secondaryArchSuffix == $portVersion base
+	devel:libxml2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:python
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sh
+	"
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool file
```

Comment:
Hmm, I did not know that. Thanks for the info and pointer.
(I though libplist needed libglib2 because I had seen it in the packaging made by Arch Linux.)

We don't have libcnary but it seems libplist has a copy of it embedded.
So you can get rid of all libglib stuff in the recipe.
Sorry for my feedback based on outdated info ;-)